public/includes/da-newsletter.php

```diff
@@ -0,0 +1,134 @@
+                                    <label for='email' class='cta'> " . __( 'Enter your email to get a 20% Coupon', 'draw-attention' ) . "</label>
+                                </div>
+                                <img src='" . $this->plugin_directory . "/assets/images/letter.svg' alt='News Letter Image'>
+                            </div>
+
+                            <input type='hidden' name='u' value='65E1000B4D683' data-name='u'>
+                            <input type='hidden' name='f' value='1' data-name='f'>
+                            <input type='hidden' name='s' data-name='s'>
+                            <input type='hidden' name='c' value='0' data-name='c'>
+                            <input type='hidden' name='m' value='0' data-name='m'>
+                            <input type='hidden' name='act' value='sub' data-name='act'>
+                            <input type='hidden' name='v' value='2' data-name='v'>
+                            <input type='hidden' name='or' value='9f41f9016dc2e4b2b014589da6eb4bad' data-name='or'>
+
+                            <div class='_form_element _x45964534 _full_width input-field'>
+                                <div class='input-field-container'>
+                                    <div class='_button-wrapper _full_width'><button id='_form_1_submit' class='_submit' type='submit'><span>" . __( 'SUBSCRIBE', 'draw-attention' ) . "</span> </button></div>
+                                    <input type='text' id='email' name='email' placeholder='Your Email' required='' aria-describedby='error-message' data-name='email'>
+                                </div>
+                            </div>
+                            
+                            <div class='error-message-container'>
+                                <p class='error-message' id='error-message'>Email address is required</p>
+                            </div>
+
+                            <div class='_form-thank-you' style='display:none;'></div>
+                            
+                            <div class='content-notice md-content-notice'>
+                                <span>" . __( "Your email is safe with us, we don't spam.", 'draw-attention' ) . '</span>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        ';
+	}
+
+	public function add_newsletter_widget() {
+		add_meta_box( 'DrawAttention_Newsletter', __( 'News Letter', 'draw-attention' ), array( $this, 'metabox_newsletter_component' ), $this->parent->cpt->post_type, 'side', 'low' );
+	}
+
+	/**
+	 * Modify the order of meta boxes for a specific post type.
+	 *
+	 * @param string $post_type The post type to modify meta box order for.
+	 */
+	function set_meta_boxes_position( $post_type ) {
+
+		global $wp_meta_boxes;
+
+		if ( 'da_image' == $post_type ) {
```

Comment:
@Majed-Habli https://www.loom.com/share/a2e3b43dcdb94a3e9c39a2af542348fc

Reply:
Thank you @tylerdigital for the explanation! i'll start making it a habit to do that instead 👍.